DustRiders/DustRiders/RoadObjectHandler.cpp

```diff
@@ -115,10 +115,10 @@ void RoadObjectHandler::addObstacles(float minDistance, float maxDistance) {
 		case ObstacleRock:
 			(*ecs)[name] = new Obstacle(name,
 				roadObjectModel,
-				ShaderProvider::carShader,
+				ShaderProvider::mapShader,
 				glm::vec3(1.f),
 				physics,
-				PxVec3(x, -0.425f, (obstacleCounter % 5 == 0) ? dist + z : dist - z),
+				PxVec3(x, -0.f,(obstacleCounter % 5 == 0) ? dist + z : dist - z),
```

Comment:
Steps currently feel too punishing using this, we might need to change the model slightly